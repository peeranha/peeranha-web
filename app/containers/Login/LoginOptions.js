import React from 'react';
import PropTypes from 'prop-types';
import { FormattedMessage } from 'react-intl';

import messages from './messages';

const LoginOptions = props => (
  <div>
    <button
      className="btn btn-secondary w-100 py-3"
      onClick={props.continueLogin}
    >
      <FormattedMessage {...messages.loginWithScatter} />
    </button>
    <hr />
    <p className="mb-0">
      <FormattedMessage {...messages.doNotHaveAcc} />
      <button className="btn btn-link" onClick={props.backToOptions}>
        <FormattedMessage {...messages.signUp} />
      </button>
    </p>
  </div>
);

LoginOptions.propTypes = {
  continueLogin: PropTypes.func,
  backToOptions: PropTypes.func,
};

export default LoginOptions;
