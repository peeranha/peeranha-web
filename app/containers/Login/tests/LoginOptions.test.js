import LoginOptions from '../LoginOptions';

it('LoginOptions', () => {
  expect(LoginOptions({})).toMatchSnapshot();
});
