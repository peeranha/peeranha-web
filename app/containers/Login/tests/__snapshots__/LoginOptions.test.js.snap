// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginOptions 1`] = `
<div>
  <button
    className="btn btn-secondary w-100 py-3"
  >
    <FormattedMessage
      defaultMessage="Log in with Scatter"
      id="app.containers.Login.loginWithScatter"
      values={Object {}}
    />
  </button>
  <hr />
  <p
    className="mb-0"
  >
    <FormattedMessage
      defaultMessage="Do not have an account?"
      id="app.containers.Login.doNotHaveAcc"
      values={Object {}}
    />
    <button
      className="btn btn-link"
    >
      <FormattedMessage
        defaultMessage="Sign up"
        id="app.containers.Login.signUp"
        values={Object {}}
      />
    </button>
  </p>
</div>
`;
